tekton/cronjobs/ko-gcloud-image-nightly-build-cron/README.md

```diff
@@ -0,0 +1,2 @@
+Cron Job to build a container image with `ko` and `gcloud` installed.
+The image is published daily to [gcr.io/tekton-releases/dogfooding/ko:gcloud-latest](gcr.io/tekton-releases/dogfooding/ko:gcloud-latest).
```

Comment:
i wonder if it might be more discoverable to have one doc that explains all the "utility" images that we are building and publishing each night :thinking:

Reply:
Yeah, it might be. Having the README.md in the folder "ensures" that when a new image is added, by copy-pasting the folder of another image, a new README is added :P 
But I'm also happy to add a new images.md.

Reply:
I updated `cronjobs/README.md` with a list of:
* all the images we build today and link to respective README for details
* all the projects with nightly release today and link to respective README for details
* all the resources with CD today and link to respective README for details